e2e_playwright/st_form_test.py

```diff
@@ -22,6 +22,17 @@
 )
 
 
+def test_form_input_performance(app: Page):
```

Comment:
I think we're still early here and I'd like to get more usage data from people before making a call. My current thinking is:
1. Existing tests are great for a baseline of performance measurements
2. But, existing tests might not cover some situations with performance issues
3. Therefore, I think it's expected for folks to add perf-specific tests if they are looking at making targeted performance improvements

In the short-term, just getting perf data and getting people into the motion of leveraging this information is the main goal. Long-term, it may open up other questions about a performance-specific test suite or running them separately.